src/modules/chat/index.js

```diff
@@ -224,7 +224,7 @@ class ChatModule {
             $element.hide();
         }
 
-        const $modIcons = $element.find('.mod_icon');
+        const $modIcons = $element.find('.mod-icon').parent();
```

Comment:
this is dangerous, so i cannot commit this. if parent is ever the message itself we could accidentally remove the entire message.